app/models/observation.rb

```diff
@@ -345,12 +345,59 @@ def owners_votes
   # Get a list of this User's Votes for this Observation.
   def users_votes(user)
     result = []
-    for n in namings
+    namings.each do |n|
       if v = n.users_vote(user)
         result << v
       end
     end
-    return result
+    result
+  end
+
+  # list of Owners's "favorite" votes for this Observation
```

Comment:
I'd remove most of the one line comments in this section.  The names say the same thing and we're guaranteed they are correct.  Chapter 4 in book "Clean Code" (PDF here: http://vitoex.googlecode.com/svn/trunk/Read/Clean%20Code.pdf) is what crystalized my current views on comments.